.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
Now you are mixing things up. We want both a package-ecosystem for "github-actions" and one for "cargo".